modulefiles/build.orion

```diff
@@ -21,7 +21,7 @@ module load nemsiogfs/2.5.3
 module load landsfcutil/2.4.1
```

Comment:
@JeffBeck-NOAA  Had to change the first line to #%Module### to get it to compile on Orion.

Reply:
@GeorgeGayno-NOAA, I pushed a commit to update the build.orion file.